src/v2/Apps/_About2/AboutApp.tsx

```diff
@@ -181,7 +184,7 @@ const AppDownload: React.FC = () => {
             rel="noopener noreferrer"
           >
             <Image
-              src="http://files.artsy.net/images/download-ios-app-transparent.svg"
+              src="http://files.artsy.net/images/download-ios-app.svg"
               mr={4}
```

Comment:
Non-blocking: Just curious @dzucconi I thought in general we do not apply margin spacing directly on palette <image>?

Reply:
It's fine to do so. Just sometimes `Spacer` components are more clear. Kind of depends on the context, I guess.

Reply:
@The-Beez-Kneez - You might be thinking of the rule around how we don't apply margin spacing on outer-most container components?